python/feldera/rest/feldera_client.py

```diff
@@ -845,6 +846,7 @@ def listen_to_pipeline(
             "json" format, the default value is False
 
         :param timeout: The amount of time in seconds to listen to the stream for
+        :param case_sensitive: True if the table name is case sensitive, False by default

```

Comment:
this may be true for column names too, is that handled anywhere

Reply:
We don't need to explicitly handle column names as they just get returned as a key in the dict.